CameraServer/Controllers/CameraController.cs

```diff
@@ -28,14 +30,39 @@ public async Task<ObjectResult> Hello()
         public async Task<FileContentResult> GetImage()
         {
             MockedImageProvider imageProvider = new MockedImageProvider();
-            return CameraImage.GetResponse(await imageProvider.GetImage());
+            return (await imageProvider.GetImage()).ToResponse();
         }
 
         [HttpGet("image")]
         public async Task<FileContentResult> GetCameraImage()
         {
             LocalCameraImageProvider imageProvider = new LocalCameraImageProvider();
-            return CameraImage.GetResponse(await imageProvider.GetImage());
+            return (await imageProvider.GetImage()).ToResponse();
+        }
+
+        [HttpPost("update-image")]
+        public async Task<ObjectResult> UpdateCameraImage([FromForm] IFormFile image, [FromForm] int cameraId)
+        {
+            if (cameraId < 1)
+                return new ApiResponse("Invalid id in FormData", HttpStatusCode.BadRequest);
+
+            using (Stream stream = image.OpenReadStream())
+            {
+                using (MemoryStream memoryStream = new MemoryStream((int)stream.Length))
+                {
+                    await stream.CopyToAsync(memoryStream);
+                    await CameraContainer.Instance.SetImage(cameraId , new CameraImage(memoryStream.ToArray()));
```

Comment:
ta in memoryStream här istället, det blir snyggare. Konstruktorn kan anropa ToArray()